.github/workflows/do_not_merge.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Do Not Merge
+
+# action to block merging on specific labels
+on:
+  pull_request:
+    types: [synchronize, opened, reopened, labeled, unlabeled]
+
+permissions:
+  checks: write
```

Comment:
I _think_ we shouldn't need this permission either, as it's for creating Check Runs with the API as an App. In this case, the workflow runner itself is the App, and should already have permissions to write a status. _If that's correct_, we can get by with `permissions: none`.